bin/kafka-producer-perf-test.sh

```diff
@@ -17,4 +17,4 @@
 if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
-exec $(dirname $0)/kafka-run-class.sh kafka.tools.ProducerPerformance $@
+exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.clients.tools.ProducerPerformance $@
```

Comment:
The old ProducerPerformance mixes performance testing and system testing together. The logic is now split into  VerifiableProducer and the new ProducerPerformance. So the command line won't be compatible.